docs/guide/controller-programming.md

```diff
@@ -113,35 +296,119 @@ Hence accessing the array beyond index 2 is illegal and may crash the controller
 Finally, note that the array elements should not be modified, for this reason the pointer is declared as *const*.
 Here are correct examples of code using these functions:
 
+%tab-component "language"
+%tab "C"
 ```c
-const double *pos = wb_gps_get_values(gps);
+const double *values = wb_gps_get_values(gps);
 
-// OK, to read the values they should never be explicitly deleted by the controller code.
-printf("MY_ROBOT is at position: %g %g %g\n", pos[0], pos[1], pos[2]);
+// OK, to read the values they should never be explicitly deleted by the controller code
+printf("MY_ROBOT is at position: %g %g %g\n", values[0], values[1], values[2]);
 
 // OK, to copy the values
 double x, y, z;
-x = pos[0];
-y = pos[1];
-z = pos[2];
+x = values[0];
+y = values[1];
+z = values[2];
 
 // OK, another way to copy the values
-double a[3] = { pos[0], pos[1], pos[2] };
+double a[3] = { values[0], values[1], values[2] };
 
 // OK, yet another way to copy these values
 double b[3];
-memcpy(b, pos, sizeof(b));
+memcpy(b, values, sizeof(b));
```

Comment:
I believe this is overkill. We should remove it.

Reply:
You mean all the "extra ways" to copy as well as their equivalent in other languages ?

Reply:
Yes. I think it doesn't bring anything helpful, right?

Reply:
Yes I never thought so while translating them... A copy is just a copy in the end